src/js/Content/Features/Store/Common/FDRMWarnings.js

```diff
@@ -34,7 +34,8 @@ export default class FDRMWarnings extends Feature {
 
         // Ubisoft Uplay detection
         const uplay
-                = text.includes("uplay")
+                = text.includes("account:\xa0uplay") // this is a &nbsp;
+            || text.includes("uplay account")
```

Comment:
Wow ok I never thought a dev called "uplay" would appear... but this change has some issues: the "account" part in DRM notices is localized, so it will only work when viewing the store in English, and "uplay account" won't match the text on some sub pages, like https://store.steampowered.com/sub/30265/ (this is a very rare case, I admit).

I think the problem is caused by using the text in game details, which contains developer/publisher names and other app metadata to match potential DRM on app pages. This can be solved by only using the text in game details when on bundle/sub pages. I've created #1344 to fix it this way.